linkis-computation-governance/linkis-client/linkis-computation-client/src/test/scala/com/webank/wedatasphere/linkis/ujes/UJESClientImplBigResultSetDemoNew.scala

```diff
@@ -0,0 +1,101 @@
+package com.webank.wedatasphere.linkis.ujes
+
+import com.google.gson.Gson
+import com.webank.wedatasphere.linkis.common.utils.Utils
+import com.webank.wedatasphere.linkis.httpclient.dws.authentication.{StaticAuthenticationStrategy, TokenAuthenticationStrategy}
+import com.webank.wedatasphere.linkis.httpclient.dws.config.{DWSClientConfig, DWSClientConfigBuilder}
+import com.webank.wedatasphere.linkis.manager.label.constant.LabelKeyConstant
+import com.webank.wedatasphere.linkis.protocol.constants.TaskConstant
+import com.webank.wedatasphere.linkis.ujes.client.UJESClient
+import com.webank.wedatasphere.linkis.ujes.client.request.JobExecuteAction.EngineType
+import com.webank.wedatasphere.linkis.ujes.client.request.{JobExecuteAction, JobSubmitAction, ResultSetAction}
+import com.webank.wedatasphere.linkis.ujes.client.response.JobExecuteResult
+import org.apache.commons.io.IOUtils
+
+import java.util.concurrent.TimeUnit
+import java.util
+
+
+object UJESClientImplBigResultSetDemoNew {
+
+
+  def main(args: Array[String]): Unit = {
+    val executeCode = "show tables"
+    val user = "hadoop"
+
+    // 1. 配置DWSClientBuilder，通过DWSClientBuilder获取一个DWSClientConfig
```

Comment:
Please modify all of Chinese to English.

Reply:
done, thx